nimare/decode/continuous.py

```diff
@@ -238,13 +238,20 @@ class CorrelationDistributionDecoder(Decoder):
     evaluate results based on significance.
     """
 
+    @use_memmap(LGR, n_files=1)
     def __init__(
-        self, feature_group=None, features=None, frequency_threshold=0.001, target_image="z"
+        self,
+        feature_group=None,
+        features=None,
+        frequency_threshold=0.001,
+        target_image="z",
+        memory_limit="1gb",
```

Comment:
Maybe we should drop `low_memory` completely and replace with `memory_limit`. `memory_limit=None` == `low_memory=False`.

Reply:
Done in 0892e75.